File: talentboard-service/src/main/java/application/models/Job.java
package application.models;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Document
public class Job {

    @Id
    private String id;
    private String title;
    private List<String> columns;
    private List<String> applicantIds;

    public Job() {}

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public List<String> getColumns() {
        return columns;
    }

    public List<String> getApplicantIds() {
        return applicantIds;
    }
}

File: talentboard-app/src/app/model/job.ts
export class Job {

  public id: string;
  public title: string;
  public columns: Array<string>;
  public applicantIds: Array<string>;

  constructor(title: string, columns: Array<string>, applicantIds: Array<string>) {
    this.title = title;
    this.columns = columns;
    this.applicantIds = applicantIds;
  }
}
